.github/workflows/ci.yaml

```diff
@@ -64,6 +64,9 @@ jobs:
           fetch-depth: 0
       - uses: helm/chart-testing-action@v2.2.1
 
+      - name: Add newrelic repository
+        run: helm repo add newrelic https://helm-charts.newrelic.com
+
```

Comment:
Do we really need this? Every chart installed during linting/integration is local. This is the only pipeline where this is failing :/

Reply:
I am able to reproduce it and I don't know why this happens... Maybe another Helm issue.